### PR TITLE
update models.py

### PR DESCRIPTION
**Pull request by krishnanjali25**

### CODE REVIEW

**Comment by arshadsid** [comment]
Are you running isort on your master or health_center branch?

**Comment by arshadsid** [comment]
I ran after cloning it from your branch and I got the same error as shown in travis.

**Comment by krishnanjali25** [comment]
i am running isort on health_center branch

**Comment by krishnanjali25** [comment]
how to resolve this issue @Samy-33

**Comment by arshadsid** [comment]
check your repo I made a commit

**Comment by arshadsid** [comment]
@krishnanjali25 close older PRs

**Comment by krishnanjali25** [comment]
@arshadsid  thank you for helping me out

File: FusionIIIT/applications/health_center/models.py [hunk 1]
```diff
@@ -1,3 +1,113 @@
...
+            ('15', '3 p.m.'),
+            ('16', '4 p.m.'),
+            ('17', '5 p.m.'),
+            ('18', '6 p.m.'),
+            ('19', '7 p.m.'),
+            ('20', '8 p.m.'),
+            ('21', '9 p.m.'),
+            ('22', '10 p.m.'),
+            ('23', '11 p.m.'),
+            )
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/models.py:45]: Is this tuple of tuples required? @krishnanjali25

**Pull request merged**